packages/salesforcedx-vscode-core/src/telemetry/telemetryReporter.ts

```diff
@@ -130,6 +130,10 @@ export default class TelemetryReporter extends vscode.Disposable {
       commonProperties['common.vscodemachineid'] = vscode.env.machineId;
       commonProperties['common.vscodesessionid'] = vscode.env.sessionId;
       commonProperties['common.vscodeversion'] = vscode.version;
+      if (vscode.env.uiKind) {
+        commonProperties['common.vscodeuikind'] =
+          vscode.UIKind[vscode.env.uiKind];
+      }
```

Comment:
Logging enum name ("Desktop" or "Web") rather than value (1 or 2) because it is more meaningful.